src/services/suite.service.js

```diff
@@ -14,26 +20,23 @@ const tasksQuery = gql`query {
 `;
 
 export const SuiteService = {
-  createGraphqlClient(host, port) {
-    return createApolloClient(`http://${host}:${port}/graphql`);
+  createGraphqlClient() {
+    return createApolloClient(`${window.location.pathname}/graphql`);
   },
   getSuites() {
-    return store.dispatch('suites/setSuites', []).then(() => {
-      return axios.get(window.location.pathname + '/suites').then((response) => {
-        const suites = [];
-        for (let i = 0; i < response.data.length; i++) {
-          const entry = response.data[i];
-          suites.push(new Suite(entry.name, entry.user, entry.host, entry.port));
-        }
-        return store.dispatch('suites/setSuites', suites);
-      }).catch((error) => {
-        const alert = new Alert(error.response.statusText, null, 'error');
-        return store.dispatch('addAlert', alert);
-      });
+    const apolloClient =  this.createGraphqlClient();
+    return apolloClient.query({
+      query: suitesQuery
+    }).then((response) => {
+      const suites = response.data.workflows;
+      return store.dispatch('suites/setSuites', suites);
+    }).catch((error) => {
+      const alert = new Alert(error.message, null, 'error');
+      return store.dispatch('addAlert', alert);
```

Comment:
Here is the change to query from GraphQL, removing the query to REST (I copied the existing call to retrieve tasks, which is broken, but will be fixed in #70 )